src/config/ConfigManager.cpp

```diff
@@ -457,11 +457,38 @@ void CConfigManager::configSetValueSafe(const std::string& COMMAND, const std::s
             parseError = "Error setting value <" + VALUE + "> for field <" + COMMAND + ">.";
         }
     } else if (CONFIGENTRY->strValue != "") {
+        std::string newValue = VALUE.empty() ? STRVAL_EMPTY : VALUE;
```

Comment:
this should not be done in here. It belongs better to `loadConfigLoadVars` (look at the end of the func)

Reply:
Excuse my dumbness once again, but I don't see how. `loadConfigLoadVars` is not called at all when you dynamically update an option via `hyprctl keyword` (these issues exist only here, normal config reload is fine).

Reply:
you can make it a func and call it at the end of loadConfigLoadVars and at the end if dynamically parsing the keyword of parseKeyword

Reply:
I revisited this, and I would rather pass on these changes. I reverted them.